### PR TITLE
test: stable and speed up one test case

### PR DESCRIPTION
**Pull request by cfzjywxk**
### What problem does this PR solve?

Try to solve the execution [fail](https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_unit_test/detail/tidb_ghpr_unit_test/28203/pipeline) in CI unit test 
```
[2020-03-19T08:41:46.100Z] ----------------------------------------------------------------------
[2020-03-19T08:41:46.100Z] FAIL: 2pc_test.go:670: testCommitterSuite.TestAcquireFalseTimeoutLock
[2020-03-19T08:41:46.100Z] 
[2020-03-19T08:41:46.100Z] 2pc_test.go:712:
[2020-03-19T08:41:46.100Z]     // it should return after about 300ms
[2020-03-19T08:41:46.100Z]     c.Assert(elapsed, Less, 350*time.Millisecond)
[2020-03-19T08:41:46.100Z] ... compare_one time.Duration = 622795225 ("622.795225ms")
[2020-03-19T08:41:46.100Z] ... compare_two time.Duration = 350000000 ("350ms")
[2020-03-19T08:41:46.100Z] 
[2020-03-19T08:41:46.100Z] 
[2020-03-19T08:41:46.100Z] ----------------------------------------------------------------------
```

### Check List 

Tests 

- Unit test

Side effects

### CODE REVIEW

File: store/tikv/2pc_test.go [hunk 1]
```diff
@@ -668,6 +668,9 @@ func (s *testCommitterSuite) TestElapsedTTL(c *C) {
 // TestAcquireFalseTimeoutLock tests acquiring a key which is a secondary key of another transaction.
 // The lock's own TTL is expired but the primary key is still alive due to heartbeats.
 func (s *testCommitterSuite) TestAcquireFalseTimeoutLock(c *C) {
+	atomic.StoreUint64(&ManagedLockTTL, 1000)       // 1s
```
Comment (jackysp) [inline store/tikv/2pc_test.go]: Will it affect other cases?

Comment (cfzjywxk) [reply store/tikv/2pc_test.go]: `testCommitterSuite` is a `SerialSuites` suite, and `c.Parallel()` is not used in the test case method, I think it's ok

Comment (jackysp) [reply store/tikv/2pc_test.go]: Got

**Pull request merged**